src/common/common.css

```diff
@@ -71,6 +71,12 @@ body {
 .success::before {
   background-image: url('/common/img/check.svg');
 }
+.icon-dismiss {
+  background-image: url('/common/img/close.svg');
+  position: absolute;
+  right: 10px;
```

Comment:
this "margin" should be 4px, see https://design.firefox.com/photon/components/message-bars.html#sizes-and-grid
but remember that the button/icon itself also already has a margin…

Reply:
Correction: See https://github.com/rugk/offline-qr-code/pull/64#issuecomment-385461872